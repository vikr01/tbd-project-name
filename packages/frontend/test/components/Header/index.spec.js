import React from 'react';
import renderer from 'react-test-renderer';
import Header from 'tbd-frontend-name/src/components/Header';

describe('Header', () => {
  test('has nothing for now', () => {
    const header = <Header />;
    const tree = renderer.create(header);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
