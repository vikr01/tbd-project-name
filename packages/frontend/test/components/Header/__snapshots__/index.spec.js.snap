// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header has nothing for now 1`] = `null`;
